apps/wizard/installationpage.cpp

```diff
@@ -154,6 +154,14 @@ void Wizard::InstallationPage::showFileDialog(Wizard::Component component)
         name = QLatin1String("Bloodmoon");
         break;
     }
+    logTextEdit->appendHtml(tr("<p>Attempting to install component %1.</p>").arg(name));
+    mWizard->addLogText(tr("Attempting to install component %1.").arg(name));
+
+    QMessageBox msgBox;
+    msgBox.setWindowTitle(tr("%1 Installation").arg(name));
+    msgBox.setIcon(QMessageBox::Information);
+    msgBox.setText(QObject::tr("Select a <b>valid</b> %1 installation media.").arg(name));
```

Comment:
I think, you need explain, what mean "valid", especially as it emphasised. This may look like a mockery of a user who tries to specify a path but doesn't understand what exactly to specify

Reply:
Is this better?
![изображение](https://user-images.githubusercontent.com/21265616/73077100-8c870500-3ed0-11ea-995d-34cc73291132.png)

Reply:
Ideally, If you can specify exactly file(s), that is looked for by wizard, for example "folder, where morrowind.exe is located" or something. If indication of cab files is enough, then good (I don't known morrowind directory layout)

Reply:
It's not a folder where Morrowind is located (which is completely unrelated). It's the actual Morrowind CD.